actions/image.list.yaml

```diff
@@ -0,0 +1,85 @@
+---
+description: List images
+enabled: true
+entry_point: src/image_actions.py
+name: image.list
+parameters:
+  timeout:
+    default: 99999
```

Comment:
The timeout is a tad too long, we should cap it to something sensible like `3600` seconds (1 hour) or so

Reply:
1.5 hours? - I have seen 45 minutes before, it may take longer in the future